cpp/unreal_plugins/SimulationController/Source/SimulationController/ImitationLearningTask.cpp

```diff
@@ -9,8 +9,6 @@
 #include <string>
 #include <vector>
 
-#include <AI/Navigation/NavigationTypes.h>
-#include <AI/NavigationSystemBase.h>
 #include <Components/SceneComponent.h>
 #include <Delegates/IDelegateInstance.h>
 #include <DrawDebugHelpers.h>
```

Comment:
It is good that you have removed some headers but I think there are more that can be removed, e.g., `DebugDrawHelpers.h`.

Reply:
I think I've now removed all the unwanted headers.